src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/UnsafeNativeMethods.cs

```diff
@@ -445,7 +445,7 @@ internal sealed class LSA_FOREST_TRUST_DOMAIN_INFO
     }
 
     [StructLayout(LayoutKind.Sequential)]
```

Comment:
Nit: Now that all these types are `struct`s we can remove `[StructLayout(LayoutKind.Sequential)]`.